src/App.test.tsx

```diff
@@ -1,9 +1,33 @@
-import { render, screen } from "@testing-library/react"
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+import { cleanup, render, screen, waitFor } from "@testing-library/react"
 import App from "./App"
 import React from "react"
+import { usePlayers } from "./hooks/api/usePlayers"
+
+jest.mock("./hooks/api/usePlayers")
+
+const usePlayersMock = jest.mocked(usePlayers)
+
+beforeEach(() => {
+  jest.resetAllMocks()
+})
+
+afterEach(cleanup)
+
+const usePlayersReturn = jest.mocked({
```

Comment:
only mock the used functions, that the test does not break when we add a new function to the usePlayers hook